beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -380,6 +381,94 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         Ok(())
     }
 
+    pub async fn handle_light_client_updates_by_range(
+        self: &Arc<Self>,
+        peer_id: PeerId,
+        request_id: PeerRequestId,
+        request: LightClientUpdatesByRangeRequest,
+    ) {
+        self.terminate_response_stream(
+            peer_id,
+            request_id,
+            self.clone()
+                .handle_light_client_updates_by_range_request_inner(peer_id, request_id, request)
+                .await,
+            Response::BlocksByRange,
+        );
+    }
+
+    /// Handle a `LightClientUpdatesByRange` request from the peer.
+    pub async fn handle_light_client_updates_by_range_request_inner(
+        self: Arc<Self>,
+        peer_id: PeerId,
+        request_id: PeerRequestId,
+        req: LightClientUpdatesByRangeRequest,
+    ) -> Result<(), (RPCResponseErrorCode, &'static str)> {
+        debug!(self.log, "Received LightClientUpdatesByRange Request";
+            "peer_id" => %peer_id,
+            "count" => req.count,
+            "start_period" => req.start_period,
+        );
+
+        // TODO(lc-update) use correct val here
+        // Should not send more than max light client updates
+        let max_request_size: u64 = 1;
```

Comment:
1 is definitely too low

Reply:
Use `MAX_REQUEST_LIGHT_CLIENT_UPDATES` defined as a constant here. It's not in the config or spec so we have to hardcode here.